src/snowflake/cli/plugins/snowpark/commands.py

```diff
@@ -304,9 +305,21 @@ def _deploy_single_object(
     }
 
 
-def _get_snowpark_artifact_path(snowpark_definition: Snowpark):
-    source = Path(snowpark_definition.src)
-    artifact_file = Path.cwd() / (source.name + ".zip")
+def _get_snowpark_source_absolute_path(
+    project_root: Path, snowpark_definition: Snowpark
+) -> Path:
+    defined_source_path = Path(snowpark_definition.src)
```

Comment:
```suggestion
    project_root: Path, defined_source_path: Path
) -> Path:
```
To avoid pushing whole pd to utility function. WDYT?

Reply:
these methods are now part of `SnowparkPackagePaths`